test/03-date-tests.js

```diff
@@ -83,10 +83,6 @@ describe('03-date-tasks', function() {
                 startDate: new Date(2000,1,1,10,0,0),
                 endDate:   new Date(2000,1,1,15,20,10,453),
                 expected:  '05:20:10.453'
-            }, {
-                startDate: new Date(2000,1,1,10,0,0),
-                endDate:   new Date(2000,1,2,15,20,10,453),
-                expected:  '29:20:10.453'
             }
```

Comment:
WOW! Please revert it. All tests are correct 
https://www.youtube.com/watch?v=rR4n-0KYeKQ

Reply:
> WOW! Please revert it. All tests are correct
> https://www.youtube.com/watch?v=rR4n-0KYeKQ

hello Ilya, I revert tesk3 , and edit task3, npm test is positive, but travic-ci is failed, why?

Reply:
i creat pr task5-9 and similar failed from travic-ci

Reply:
Please provide me answers to my questions. i look to hearing from you soon